documenting.rst

```diff
@@ -1643,9 +1643,10 @@ in production, other are work in progress:
 | Lithuanian   |                    | `mail <https://mail.python.org/pipermail/|
 | (lt)         |                    | doc-sig/2019-July/004138.html>`__        |
 +--------------+--------------------+------------------------------------------+
-| Polish (pl)  |                    | `mail <https://mail.python.org/pipermail/|
-|              |                    | doc-sig/2019-April/004106.html>`__       |
-|              |                    |                                          |
+| Polish (pl)  | `Maciej Olko       | `transifex <https://www.transifex.com/   |
```

Comment:
The URL https://www.transifex.com/python-doc/python-newest/language/pl/ is 404 for me

Reply:
You have to be logged in to Transifex to see this page.

Please not that ``transifex`` link for Simplified Chinese behave the same way.